src/vs/workbench/parts/extensions/electron-browser/extensionTipsService.ts

```diff
@@ -326,6 +339,45 @@ export class ExtensionTipsService extends Disposable implements IExtensionTipsSe
 							this.telemetryService.publicLog('extensionRecommendations:popup', { userReaction: 'cancelled', extensionId: name });
 						});
 					});
+
+				const fileExtensionSuggestionIgnoreList = <string[]>JSON.parse(this.storageService.get
+					('extensionsAssistant/fileExtensionsSuggestion', StorageScope.GLOBAL, '[]'));
+
+				if (!hasSuggestion &&
+					fileExtension &&
+					fileExtensionSuggestionIgnoreList.indexOf(fileExtension) === -1 &&
+					mimeTypes.length === 1 &&
+					mimeTypes[0] === MIME_UNKNOWN) {
+
+					let message = localize('showLanguageExtensions', "Search Marketplace Extensions for '{0}'...", fileExtension);
+
+					const searchMarketplaceAction = this.instantiationService.createInstance(ShowLanguageExtensionsAction, fileExtension);
+
+					const options = [
+						localize('ok', "OK"),
+						localize('neverShowAgain', "Don't show again"),
+						localize('close', "Close")
+					];
+
+					this.choiceService.choose(Severity.Info, message, options, 3).done(choice => {
+						switch (choice) {
+							case 0:
+								return searchMarketplaceAction.run();
```

Comment:
Can you add the telemetry events to track what the user does here just like in the other `choiceService`?